src/page/Page.tsx

```diff
@@ -179,13 +180,13 @@ export const NeoPage = ({
           onPageLayoutUpdate(newLayout);
         }}
       >
-        {reports.map((report, index) => {
+        {reports.map((report) => {
           const w = 12;
           // @ts-ignore
           return (
             <Grid
-              index={getReportIndex(pagenumber, index)}
-              key={getReportIndex(pagenumber, index)}
+              index={getReportIndex(pagenumber, report.index)}
```

Comment:
maybe this is where it gets confused, as it's not sure how to handle the generated ID?

Reply:
Nope, it was SettingsThunks L 20 (hardResetCardSettings dispatch)